### PR TITLE
add RUN_OPTS support for systemd

### PR DESCRIPTION
**Pull request by kardapoltsev**
RUN_OPTS could be specified in `/etc/defalt/app_name` like in a `SystemV` setup

### CODE REVIEW

File: src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemd/start-template [hunk 1]
```diff
@@ -6,7 +6,7 @@ Requires=${{start_facilities}}
 Type=simple
 WorkingDirectory=${{chdir}}
 EnvironmentFile=${{env_config}}
-ExecStart=${{chdir}}/bin/${{exec}}
+ExecStart=${{chdir}}/bin/${{exec}} $RUN_OPTS
```
Comment (muuki88) [inline src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemd/start-template]: This should already work. The `EnvironmentFile` above provides the ability to add environment variables.

In systemv this always felt like a hack.

Comment (muuki88) [inline src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemd/start-template]: I took a look at the code. Currently only the debian script supports this. Using global variables for application startup seems insecure and magic. 

What is the problem you are trying to solve?

Comment (kardapoltsev) [inline src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemd/start-template]: I'm trying to pass command line arguments to my app.

Comment (kardapoltsev) [inline src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemd/start-template]: > I'd rather remove the RUN_OPTS support from debian and improve the application configuration documentation
Let's go this way (:

**Comment by muuki88** [comment]
Okay cool :) Can I help you with your specific problem?

**Pull request closed without merging**